terraform/staging/main.tf

```diff
@@ -184,11 +184,11 @@ module "certificates" {
 module "database" {
   source = "../modules/rds"
 
-  allocated_storage         = 25
+  allocated_storage         = 100
   apply_changes_immediately = true
   backup_retention_period   = 7
   highly_available          = false
-  instance_class            = "db.t3.micro"
+  instance_class            = "db.t3.small"

```

Comment:
We might retain this instance class for staging down the line if the cost save is negligible